<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61, 167, 123, 255), stop:1 rgba(196, 255, 192, 255))</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>680</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" rowspan="10">
         <widget class="QTextBrowser" name="display_terminal">
          <property name="styleSheet">
           <string notr="true">background-color : black; color : white</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLineEdit" name="terminal">
          <property name="styleSheet">
           <string notr="true">background-color : black; color : white</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(20, 141, 58,123);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>8251 : USART</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="5">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QLineEdit" name="cwb"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="bllable">
            <property name="text">
             <string>BL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>CW-A</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>CW_B</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>AL</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="dpb"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>DP-A</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>DP-B</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="al"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="cwa"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="dpa"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>CL</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="dl">
            <property name="text">
             <string>DL</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="bl"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="cl"/>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="dl_2"/>
          </item>
         </layout>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="soursebut">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:Blue</string>
          </property>
          <property name="text">
           <string>Enter Data on source address</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QRadioButton" name="blocktra">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Block Transfer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QRadioButton" name="bytetra">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Byte Transfer</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Hardware Mode</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QTextBrowser" name="status">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(92, 255, 223,0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="about">
          <property name="text">
           <string>About App</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="desbut">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:blue</string>
          </property>
          <property name="text">
           <string>See Data on Destination address</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3" rowspan="6">
         <widget class="QLabel" name="blockdig">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(92, 255, 223,0);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Developed By : Niranjan Rathod (SE-4)</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="image">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>601</width>
          <height>571</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>31</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>C
L
K</string>
        </property>
       </widget>
       <widget class="QPushButton" name="led1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="led1_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextBrowser" name="display2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="display2_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>231</width>
          <height>81</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you are unable to see Kit image on left. copy &amp;quot;dyna8251.jpg&amp;quot; from project derectory to build directory .&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLineEdit" name="terminal_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>386</y>
          <width>351</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : black; color : white</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="display_terminal_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>351</width>
          <height>371</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : black; color : white</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-- Return</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>121</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &amp;quot;E&amp;quot; command&lt;/p&gt;&lt;p&gt; to enter data.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>20</y>
          <width>561</width>
          <height>501</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4210d8;&quot;&gt;Copy All Image Files From Project Directory to build directory.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#dace1d;&quot;&gt;Best viewd in 1080p.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This application is developed to simulate the &lt;span style=&quot; color:#ff0000;&quot;&gt;Environment of&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;MIT KIT Assignment Practicals &lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;for those who haven't practiced them well in Lab.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;These Applications are rapidly developed within duration of&lt;span style=&quot; color:#00aa00;&quot;&gt; two&lt;/span&gt; days.&lt;/p&gt;&lt;p&gt;Therefore this application may not meet &lt;span style=&quot; color:#00aa00;&quot;&gt;expection&lt;/span&gt; of many users.&lt;/p&gt;&lt;p&gt;Still having something to practise is always better than having nothing.&lt;/p&gt;&lt;p&gt;Note that &lt;span style=&quot; color:#00aa00;&quot;&gt;Only 8251&lt;/span&gt; have 'Simulated Compiler' support. &lt;/p&gt;&lt;p&gt;So be Careful and type ONLY valid codes. If you are facing any problem &lt;/p&gt;&lt;p&gt;with this application &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;1]&lt;/span&gt; Please Use &lt;span style=&quot; color:#ff0000;&quot;&gt;sample code&lt;/span&gt; given in Project directoy.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;2] &lt;/span&gt;Or refer to &lt;span style=&quot; color:#ff0000;&quot;&gt;video &lt;/span&gt;given in same directory.&lt;/p&gt;&lt;p&gt;and if you like to develop things you are always &lt;span style=&quot; color:#00aa00;&quot;&gt;welcome to contribute&lt;/span&gt;&lt;/p&gt;&lt;p&gt;to repository.&lt;br/&gt;-Niranjan Rathod &lt;/p&gt;&lt;p&gt;For Details and bugs report and review : P : Whatsapp at +917276757446&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
